rules-tests/TypeDeclaration/Rector/ClassMethod/AddVoidReturnTypeWhereNoReturnRector/FixtureUsePhpDoc/never-types.php.inc

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\AddVoidReturnTypeWhereNoReturnRector\Fixture;
+
+final class SomeClass
+{
+    /**
+     * @return never
+     */
+    public function neverReturn()
+    {
+        header('Location: example.com');
+        exit();
+    }
+
+    /**
+     * @return never-return
+     */
+    public function neverReturn()
+    {
+        header('Location: example.com');
+        exit();
+    }
+
+    /**
+     * @return never-returns
+     */
+    public function neverReturns()
+    {
+        header('Location: example.com');
+        exit();
+    }
+
+    /**
+     * @return no-return
+     */
+    public function noReturn()
+    {
+        header('Location: example.com');
+        exit();
+    }
```

Comment:
it seems rector does not support all "aliases" of the bottom-type phpstan supports:

https://phpstan.org/writing-php-code/phpdoc-types#bottom-type

only `@return never` is turned into the `NeverType` ..

Reply:
fixed in the latest commit